waveorder/filter.py

```diff
@@ -0,0 +1,196 @@
+import itertools
+import numpy as np
```

Comment:
unused import

Reply:
Sidenote: we should add linting for this.

Reply:
Fixed in #182.